### PR TITLE
Add jd45 keyboard

### PR DESCRIPTION
**Pull request by matthewdias**
Pending qmk reorg

## QMK Pull Request 

https://github.com/qmk/qmk_firmware/pull/12520

## Checklist

- [ ] The VIA support for this keyboard is in QMK master already (MANDATORY)
- [x] The VIA definition follows the guide here: https://caniusevia.com/docs/layouts
- [x] I have tested this keyboard definition using VIA's "Design" tab.
- [x] I have tested this keyboard definition with firmware on a device.
- [x] I have assigned alpha keys and modifier keys with the correct colors.
- [x] The Vendor ID is not `0xFEED`

### CODE REVIEW

File: src/jd45/jd45.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{
+  "name": "JD45",
+  "vendorId": "0x6A64",
+  "productId": "0x6060",
```
Comment (wilba) [inline src/jd45/jd45.json:4]:
```suggestion
  "vendorId": "0x6582",
  "productId": "0x0004",
```

File: src/jd45/jd45.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  "lighting": "qmk_rgblight",
+  "matrix": {
+    "rows": 4,
+    "cols": 13
+  },
+  "layouts": {
+    "labels": [
+      ["Bottom Row", "Split Space", "6.25u", "6u"]
```
Comment (wilba) [inline src/jd45/jd45.json:12]:
```suggestion
      ["Bottom Row", "Split Space", "6.25U", "6U"]
```

**Review by wilba — CHANGES_REQUESTED** [review]

**Comment by wilba** [comment]
Make these changes, I'll merge now and then move it to `wilba_tech` later.
You can build your own QMK firmware with the changed VID/PID in the short term.

**Comment by matthewdias** [comment]
Done. Thanks for doing that!

**Pull request merged**